ibis/sql/postgres/tests/test_functions.py

```diff
@@ -1548,3 +1549,233 @@ def test_string_to_binary_round_trip(con):
         [row[0][0] for row in con.raw_sql(sql_string).fetchall()], name='tmp'
     )
     tm.assert_series_equal(result, expected)
+
```

Comment:
is it possible to move tests to https://github.com/ibis-project/ibis/blob/master/ibis/tests/all/test_geospatial.py ?

Reply:
These tests are running on the Postgres instance -- are the ones in `ibis/tests/all` supposed to be targeting a specific backend? I confess I don't have a great grasp on the test directory structure.

Reply:
yes, current the tests in this file just runs for mapd ... so you can use the same approach for postgres. 
and for the current tests there (mapd), if possible, you can allow it to test also for postgres.

Reply:
Alternatively, you could move these to `test_postgis.py` and write `pytestmark = pytest.mark.postgis` to avoid having to decorate every test.